### ISSUE TITLE
warning: ‘OPENNAME_NAME_UPDATE’ defined but not used

### ISSUE THREAD

**Issue by poiuty** [FeatherCoin/Feathercoin#91]
```
In file included from checkpointsync.cpp:72:0:
main.h:130:20: warning: ‘OPENNAME_NAME_PREORDER’ defined but not used [-Wunused-variable]
 static const char* OPENNAME_NAME_PREORDER="a";
                    ^
main.h:131:20: warning: ‘OPENNAME_NAME_REGISTRATION’ defined but not used [-Wunused-variable]
 static const char* OPENNAME_NAME_REGISTRATION="b";
                    ^
main.h:132:20: warning: ‘OPENNAME_NAME_UPDATE’ defined but not used [-Wunused-variable]
 static const char* OPENNAME_NAME_UPDATE="c";
                    ^
main.h:133:20: warning: ‘OPENNAME_NAME_TRANSFER’ defined but not used [-Wunused-variable]
 static const char* OPENNAME_NAME_TRANSFER="d";
                    ^
main.h:134:20: warning: ‘OPENNAME_NAME_RENEWAL’ defined but not used [-Wunused-variable]
 static const char* OPENNAME_NAME_RENEWAL="e";
```

**Comment by wellenreiter01** [comment]
This is a warning only and has no influence on the compile process to the functionality of the code or the resulting binaries.
It is due to a future implementation of Openname.

**Comment by wrapperband** [comment]
Openname is now implemented, what is the status on these variables warnings? 

Are they still needed or are superfluous? If so I will work through removing them.

**Comment by wellenreiter01** [comment]
They are definded in main.h, but not used, so I think we can comment them out and delete in 0.9.3.2

**Comment by wrapperband** [comment]
I was not able to delete the definitions from main.h  as they are used in one function. I think the problem is the way they have been defined, which is too public. I will ask Ghostlander.

**Comment by wellenreiter01** [comment]
It's no real problem anyway. It's just a warning and no error. 
W.
On Jun 26, 2016 12:41 AM, wrapperband <notifications@github.com> wrote:I was not able to delete the definitions from main.h  as they are used in one function. I think the problem is the way they have been defined, which is too public. I will ask Ghostlander.

**Comment by cqtenq** [comment]
These definitions generate warnings, but they are used in utility dialog.cpp

**Comment by wellenreiter01** [comment]
Fix available, will be applied to release 0.9.6
